OPM integration tests are flaky

```sh
FAILED tests/test_opm_integration/test_gridprop_io.py::test_restart_prop_reading - hypothesis.errors.Flaky: Inconsistent test results! Test case was Conclusion(status=Status.INTERESTING, interesting_origin=(<class 'FileNotFoundError'>, '/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/xtgeo/grid3d/_find_gridprop_in_eclrun.py', 608, (), ())) on first run but Conclusion(status=Status.INTERESTING, interesting_origin=(<class 'ValueError'>, '/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/xtgeo/grid3d/_find_gridprop_in_eclrun.py', 227, (), ())) on second
```

while OPM dumps core

```sh
flow: ./opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp:465: Opm::SimulatorReport Opm::AdaptiveTimeSteppingEbos<TypeTag>::step(const Opm::SimulatorTimer&, Solver&, bool, const std::vector<int>*) [with Solver = Opm::NonlinearSolverEbos<Opm::Properties::TTag::EclFlowOilWaterProblem, Opm::BlackoilModelEbos<Opm::Properties::TTag::EclFlowOilWaterProblem> >; TypeTag = Opm::Properties::TTag::EclFlowOilWaterProblem]: Assertion `dtEstimate > 0' failed.
[fv-az1269-522:04850] *** Process received signal ***
[fv-az1269-522:04850] Signal: Aborted (6)
[fv-az1269-522:04850] Signal code:  (-6)
[fv-az1269-522:04850] [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x42520)[0x7fd884c42520]
[fv-az1269-522:04850] [ 1] /lib/x86_64-linux-gnu/libc.so.6(pthread_kill+0x12c)[0x7fd884c969fc]
[fv-az1269-522:04850] [ 2] /lib/x86_64-linux-gnu/libc.so.6(raise+0x16)[0x7fd884c42476]
[fv-az1269-522:04850] [ 3] /lib/x86_64-linux-gnu/libc.so.6(abort+0xd3)[0x7fd884c287f3]
[fv-az1269-522:04850] [ 4] /lib/x86_64-linux-gnu/libc.so.6(+0x2871b)[0x7fd884c2871b]
[fv-az1269-522:04850] [ 5] /lib/x86_64-linux-gnu/libc.so.6(+0x39e96)[0x7fd884c39e96]
[fv-az1269-522:04850] [ 6] flow(+0x88b2ad)[0x5638bba502ad]
[fv-az1269-522:04850] [ 7] flow(+0x89060c)[0x5638bba5560c]
[fv-az1269-522:04850] [ 8] flow(+0x8915ab)[0x5638bba565ab]
[fv-az1269-522:04850] [ 9] flow(+0x88199b)[0x5638bba4699b]
[fv-az1269-522:04850] [10] flow(+0x802963)[0x5638bb9c7963]
[fv-az1269-522:04850] [11] flow(+0x142112)[0x5638bb307112]
[fv-az1269-522:04850] [12] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90)[0x7fd884c29d90]
[fv-az1269-522:04850] [13] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80)[0x7fd884c29e40]
[fv-az1269-522:04850] [14] flow(+0x15a3d5)[0x5638bb31f3d5]
[fv-az1269-522:04850] *** End of error message ***
Aborted (core dumped)
flow: ./opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp:465: Opm::SimulatorReport Opm::AdaptiveTimeSteppingEbos<TypeTag>::step(const Opm::SimulatorTimer&, Solver&, bool, const std::vector<int>*) [with Solver = Opm::NonlinearSolverEbos<Opm::Properties::TTag::EclFlowOilWaterProblem, Opm::BlackoilModelEbos<Opm::Properties::TTag::EclFlowOilWaterProblem> >; TypeTag = Opm::Properties::TTag::EclFlowOilWaterProblem]: Assertion `dtEstimate > 0' failed.
[fv-az1269-522:04888] *** Process received signal ***
[fv-az1269-522:04888] Signal: Aborted (6)
[fv-az1269-522:04888] Signal code:  (-6)
[fv-az1269-522:04888] [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x42520)[0x7f3597042520]
[fv-az1269-522:04888] [ 1] /lib/x86_64-linux-gnu/libc.so.6(pthread_kill+0x12c)[0x7f35970969fc]
[fv-az1269-522:04888] [ 2] /lib/x86_64-linux-gnu/libc.so.6(raise+0x16)[0x7f3597042476]
[fv-az1269-522:04888] [ 3] /lib/x86_64-linux-gnu/libc.so.6(abort+0xd3)[0x7f35970287f3]
[fv-az1269-522:04888] [ 4] /lib/x86_64-linux-gnu/libc.so.6(+0x2871b)[0x7f359702871b]
[fv-az1269-522:04888] [ 5] /lib/x86_64-linux-gnu/libc.so.6(+0x39e96)[0x7f3597039e96]
[fv-az1269-522:04888] [ 6] flow(+0x88b2ad)[0x5593a9a682ad]
[fv-az1269-522:04888] [ 7] flow(+0x89060c)[0x5593a9a6d60c]
[fv-az1269-522:04888] [ 8] flow(+0x8915ab)[0x5593a9a6e5ab]
[fv-az1269-522:04888] [ 9] flow(+0x88199b)[0x5593a9a5e99b]
[fv-az1269-522:04888] [10] flow(+0x802963)[0x5593a99df963]
[fv-az1269-522:04888] [11] flow(+0x142112)[0x5593a931f112]
[fv-az1269-522:04888] [12] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90)[0x7f3597029d90]
[fv-az1269-522:04888] [13] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80)[0x7f3597029e40]
[fv-az1269-522:04888] [14] flow(+0x15a3d5)[0x5593a93373d5]
[fv-az1269-522:04888] *** End of error message ***
Aborted (core dumped)
```

--- a/tests/test_opm_integration/test_gridprop_io.py
+++ b/tests/test_opm_integration/test_gridprop_io.py
@@ -178,6 +178,7 @@ unit_systems = st.sampled_from(UnitSystem)
 opm_setups = st.builds(OpmSetup, xtgeo_grids, st.booleans(), unit_systems)
 
 
+@pytest.mark.xfail(reason="OPM flaky")
 @pytest.mark.requires_opm
 @pytest.mark.usefixtures("setup_tmpdir")
 @settings(max_examples=5)
@@ -245,6 +246,7 @@ def test_init_props_reading(case):
     assert poro.date == "20000101"
 
 
+@pytest.mark.xfail(reason="OPM flaky")
 @pytest.mark.requires_opm
 @pytest.mark.usefixtures("setup_tmpdir")
 @settings(max_examples=5)

